app/views/admin/school_groups/_active_schools.html.erb

```diff
@@ -0,0 +1,13 @@
+<table class="table table-striped">
+  <% school_group.schools.active.by_name.each do |school| %>
```

Comment:
this should be visible, to be consistent with the counts that we're showing elsewhere.

active and visible schools should have completed their onboarding, so will be in this list.

Reply:
Thanks @ldodds - have pushed active to visible here now!